test/index.test.ts

```diff
@@ -1,3 +1,9 @@
-test('test tests placeholder', () => {
-  expect(true).toBeTruthy();
+import * as index from '../src';
+
+test('index.ts to have Calculator', () => {
+  expect(index).toHaveProperty('Calculator');
+});
+
+test('index.ts to have Calculator.calculate', () => {
```

Comment:
Are we also exposing `calculateMeasureReports`? Should we have a test for that too?